src/scripts/content/kantree.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+/* Features: */
+/* - Add timer to the card */
+/* - Add timer to the sub-tasks list */
+/* - Get toggl project name from kantree card */
+/* - Get toggl tags from kantree card */
+/* - Handle card view mode changes */
+console.log('Toggl Button loaded for kantree.');
+
+/* Card button */
+togglbutton.render('.card-view:not(.toggl)', {observe: true}, function (elem) {
+      var link, container = createTag('div', 'kt-card-toggl-btn btn btn-board-menu'),
+    cardRef = $('.card-view-header a.ref', elem),
+    cardId = cardRef && cardRef.textContent,
+    cardTitle = $('.card-view-header h2', elem) && $('.card-view-header h2', elem).textContent,
+    taskTitle =  cardId + ' ' + cardTitle,
```

Comment:
We can wake up with `taskTitle === 'null null'` apparently. 🙊 Is this what we really want?